RecoTracker/TrackProducer/plugins/MomentumConstraintProducer.cc

```diff
@@ -38,14 +38,13 @@ Description: <one line class summary>
 // class declaration
 //
 
-class MomentumConstraintProducer: public edm::EDProducer {
+class MomentumConstraintProducer: public edm::stream::EDProducer<> {
 public:
   explicit MomentumConstraintProducer(const edm::ParameterSet&);
-  ~MomentumConstraintProducer();
+  ~MomentumConstraintProducer() override;
```

Comment:
```suggestion
  ~MomentumConstraintProducer() override = default;
```
and remove the empty definition.